test/transforms.jl

```diff
@@ -680,6 +680,34 @@
     @test n1 == n2
   end
 
+  @testset "StdNames" begin
```

Comment:
Make sure you include tests with row tables. Take a look at other transforms for examples.